src/components/JobsContext.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, {
+  createContext,
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState
+} from 'react'
+import { JOBS_DOCTYPE } from '../doctypes'
+import CozyRealtime from 'cozy-realtime'
+import Alerter from 'cozy-ui/transpiled/react/Alerter'
+import logger from 'cozy-logger'
+import { useI18n } from 'cozy-ui/react/I18n'
+
+const log = logger.namespace('import.context')
+
+export const JobsContext = createContext({})
+
+export const useJobsContext = () => {
+  return useContext(JobsContext)
+}
+
+const JobsProvider = ({ children, client }) => {
+  const [jobsInProgress, setJobsInProgress] = useState([])
+  const { t } = useI18n()
+  const realtimeStartedRef = useRef(false)
+  const jobsInProgressRef = useRef([])
+
+  const realtime = useMemo(() => {
+    return new CozyRealtime({ client })
+  }, [client])
+
+  const handleRealtime = data => {
+    const { worker, state, message: msg } = data
+    const { current: currJobsInProgress } = jobsInProgressRef
+    const index = currJobsInProgress.findIndex(a => a.account === msg.account)
+    const exist = index !== -1
+
+    let arr = [...currJobsInProgress]
+    if (worker === 'konnector') {
+      if (state === 'running' && !exist) {
+        arr.push(msg)
+      } else if (state === 'done' && exist) {
+        arr.splice(index, 1)
+        Alerter.success(t('JobsContext.alerter-success'))
+      } else if (state === 'errored' && exist) {
+        arr.splice(index, 1)
+        Alerter.error(t('JobsContext.alerter-errored'))
+      }
```

Comment:
It would be nice if the Alerter logic was provided as callbacks to the JobsProvider, I am sure the JobsProvider can be used in other contexts like in Harvest and this behavior should be configurable by the app.